docs/chapters/sdk/using-the-sdk-to-crosscompile.rst

```diff
@@ -0,0 +1,21 @@
+Using the SDK to cross compile
+==============================
+
+Cross compiling with help of the SDK is not much different to normal compiling, there are more pitfalls but let's get on with the happy cases for now.
```

Comment:
... with the help of the SDK is not much different from normal compilation. There are ...